app/src/main/java/edu/wpi/first/shuffleboard/app/Shuffleboard.java

```diff
@@ -114,11 +108,6 @@ public void start(Stage primaryStage) throws IOException {
    * @throws IOException if the plugin directory could not be read
    */
   private void loadPluginsFromDir() throws IOException {
-    Path pluginPath = Paths.get(Storage.PLUGINS_DIR);
-    if (!Files.exists(pluginPath)) {
-      Files.createDirectories(pluginPath);
-    }
-    PluginLoader.getDefault().loadAllJarsFromDir(pluginPath);
+    PluginLoader.getDefault().loadAllJarsFromDir(Storage.getPluginPath());
```

Comment:
This whole method could just get inlined now it's just a single line of code.

Reply:
Didn't want to remove the large amount of documentation here. Down to do what you think is appropriate though.

Reply:
TBH that documentation should be in `Plugin` so third-party API consumers can see it. I shouldn't have put it here in the first place.